server/validate/email/validate.go

```diff
@@ -434,6 +440,64 @@ func (v *validator) Remove(user t.Uid, value string) error {
 	return store.Users.DelCred(user, validatorName, value)
 }
 
+// SendMail replacement
+//
+func (v *validator) Send_Mail(rcpt []string, msg []byte) error {
+
+	client, err := smtp.Dial(v.SMTPAddr + ":" + v.SMTPPort)
+	if err != nil {
+		return err
+	}
+	defer client.Close()
+	err = client.Hello(v.ServerName)
+	if err != nil {
+		return err
+	}
+	istls, _ := client.Extension("starttls")
+	if istls {
```

Comment:
`istls` is not used outside of the `if` block. Please use a construct like 
```go
if istls, _ := client.Extension("STARTTLS"); istls {
```

Reply:
fixed.